tencent-cloud/redis/create.go

```diff
@@ -0,0 +1,89 @@
+package redis
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	redis "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/redis/v20180412"
+
+	common "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common"
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/errors"
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/profile"
+)
+
+type TcloudRedis struct {
+	Region      string `json:"region"`
+	Instance    uint64 `json:"instance"`
+	ZoneId      uint64 `json:"zoneid"`
+	TypeId      uint64 `json:"typeid"`
+	MemSize     uint64 `json:"memsize"`
+	Password    string `json:"password"`
+	ProjectId   int64  `json:"projectid"`
+	Vport       uint64 `json:"vport"`
+	BillingMode int64  `json:"billingmode"`
+	Period      uint64 `json:"billperiod"`
```

Comment:
Try to keep field names the same as their JSON keys, it makes fields more predictable. Also I think you meant `billingPeriod`